index.html

```diff
@@ -48,7 +57,180 @@ <h1>I’m Nurbol<br>Glad to see<br> you!</h1>
+
+<!--1 work card-->            
+
+            <div class="works-card">
+                    <img src="worksimg/Snapshoot Portfolio@2x.png">
+                    <div class="info-block">
+                        <div class="top">
+                            <h2 id="six">Tonic</h2>
+                            <ul class="frame">
+                                <li id="client">CANOPY</li>
+                                <li id="dot"> </li>
+                                <li id="role">Back End Dev</li>
+                                <li id="dot"> </li>
+                                <li id="year">2015</li></ul>
+                        </div>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="bottom">
+                        <ul class="tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li></ul>
+                        <button class="action">See Project</button>
+                        </div>
+                    </div>
+            </div>
+
+<!--2 work card-->   
+
+            <div class="works-card">
+                <img src="worksimg\Snapshoot Portfolio@2x (1).png">
+                <div class="info-block">
+                    <div class="top">
+                        <h2 id="eight">Multi-Post<br>Stories</h2>
+                        <ul class="frame">
+                            <li id="client">CANOPY</li>
+                            <li id="dot"> </li>
+                            <li id="role">Back End Dev</li>
+                            <li id="dot"> </li>
+                            <li id="year">2015</li></ul>
+                    </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="bottom">
+                    <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li></ul>
+                    <button class="action">See Project</button>
+                </div>
+            </div>
+        </div>
+
+<!--3 work card-->
+
+<div class="works-card">
+    <img src="worksimg/Snapshoot Portfolio@2x (2).png">
+    <div class="info-block">
+        <div class="top">
+            <h2 id="six">Tonic</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+        </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="bottom">
+            <ul class="tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li></ul>
+            <button class="action">See Project</button>
```

Comment:
- [ ] How about adding some `box-shadow` when `button` is hovered on by users according to the design too? It makes your design more aesthetic and improves user experience.